packages/service-utils/src/core/authorize/index.ts

```diff
@@ -141,7 +141,7 @@ export async function authorize(
     return {
       authorized: false,
       status: 401,
-      errorMessage: "Missing clientId or secretKey.",
+      errorMessage: "Please include a client id or secret key with your request. Starting August 1st, 2023 API Keys are required to use thirdweb services. Learn more: https://blog.thirdweb.com/changelog/api-keys-to-access-thirdweb-infra",
```

Comment:
shouldn't we link to the https://thirdweb.com/create-api-key like we do in other places?